src/components/PublicityHeader/constants.ts

```diff
@@ -0,0 +1,37 @@
+import store from 'assets/tienda.svg';
+import delivery from 'assets/fast-delivery.svg';
+import payment from 'assets/payment-method.svg';
+import articulo1 from 'assets/articulo1.jpg';
+import articulo2 from 'assets/articulo2.jpg';
+
+export const SERVICES = [
+  {
+    id: 1,
+    image: store,
+    text1: 'PUNTO DE RETIRO',
+    text2: 'Mas información',
```

Comment:
Text can be moved to a constant and reused in all places

Reply:
no need, its never gonna be used again, never